connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -478,6 +507,8 @@ public static synchronized void sdkInitializeMobileConnect(
         context.registerReceiver(
                 SimCardStateChangedBroadcastReceiver.getReceiver(),
                 SimCardStateChangedBroadcastReceiver.getIntentFilter());
+
+        initializeCommonComponents();
```

Comment:
I think Goran's plan is for the app developer to either init the connectSDK (sdkInitialize) or MC SDK (sdkInitializeMobileConnect). initializeCommonComponents handles common inits for both, thus in a separate method called by the two.